src/app/content/highlights/components/HighlightStyles.tsx

```diff
@@ -134,6 +178,16 @@ export const BlueStickyNote = styled(StickyNote)`
     left: -${stickyNoteMeasures.bulletSize / 2}rem;
     background: ${stickyNoteMeasures.blue};
   }
+
+  @media print {
```

Comment:
is it really necessary to have these styles on the sticky note? if you did have to do something special for showing the empty state in print, i would say just hide the whole thing instead, there is no reason to be printing that